src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -56,26 +57,46 @@
           border: 1px solid rgba(0,40,100,.12);
           border-radius: 5px;
           background-color: #f0f3f94f;
+          margin-bottom: 20px !important;
         }
-        .container-repeatable-elements .delete-element {
+        .container-repeatable-elements .action-element {
           z-index: 99;
           position: absolute!important;
-          margin-left: -24px;
-          margin-top: 0px;
           height: 30px;
           width: 30px;
           border-radius: 15px;
           text-align: center;
           background-color: #e8ebf0!important;;
         }
+
+        .container-repeatable-elements .delete-element {
+					top: -15px;
+					right: -15px;
+        }
+
+        .container-repeatable-elements .move-element {
+					top: 15px;
+					left: -15px;
+					outline: none;
+					cursor: move;
+					line-height: 29px;
+        }
       </style>
   @endpush
 
   {{-- FIELD EXTRA JS --}}
   {{-- push things in the after_scripts section --}}
 
   @push('crud_fields_scripts')
+  		<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"></script>
       <script>
+				$( function() {
+			    $( "#sortable" ).sortable({
```

Comment:
Rather than adding the ID I think we can trigger sortable directly on the ```container-repeatable-elements``` class.

Reply:
@tabacitu Ok for this !